WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -107,6 +107,15 @@ wca.competitionsToMarkers = function(map, competitions) {
   return markers;
 };
 
+wca.renderMarkdownRequest = function(markdownContent) {
```

Comment:
Yeah I thought about this a while ago and decided to move all the code from there but I think it's something for another PR probably.